catalog/solr/catalog-solr-external-provider/src/main/java/ddf/catalog/solr/external/SolrHttpCatalogProvider.java

```diff
@@ -301,7 +305,7 @@ private boolean isServerUp(SolrServer solrServer) {
         return false;
     }
 
-    private SolrServer getServer() {
+    private SolrClient getServer() {
```

Comment:
I'm glad you decided to fix all the `Server`s that should be `Client`s because this looks really weird and confusing. :smile: